src/sentry/issues/endpoints/organization_derive_code_mappings.py

```diff
@@ -100,19 +106,18 @@ def post(self, request: Request, organization: Organization) -> Response:
         if not request.access.has_project_access(project):
             return self.respond(status=status.HTTP_403_FORBIDDEN)
 
-        repo_name = request.data.get("repoName")
-        stack_root = request.data.get("stackRoot")
-        source_root = request.data.get("sourceRoot")
-        branch = request.data.get("defaultBranch")
```

Comment:
All four values are now returned as an object in the call below: `code_mapping = get_code_mapping_from_request(request)`